compair/static/modules/answer/answer-module.js

```diff
@@ -257,11 +266,20 @@ module.controller(
 
                     if (ret.draft) {
                         Toaster.success("Answer Draft Saved", "Remember to submit your answer before the deadline.");
-                        $location.path('/course/' + $scope.courseId + '/assignment/' + $scope.assignmentId + '/answer/' + $scope.answer.id + '/edit');
-                        $location.replace();
+                        $scope.refreshOnExit = true;
                     } else {
```

Comment:
add `$scope.refreshOnExit = false;` in the else just in case they save a draft and then the final version in the same modal (just mostly a precaution for future changes mostly)